findoutlie/metrics.py

```diff
@@ -21,12 +21,13 @@ def dvars(img):
         One-dimensional array with n-1 elements, where n is the number of
         volumes in `img`.
     """
-    # Hint: remember 'axis='.  For example:
-    # In [2]: arr = np.array([[2, 3, 4], [5, 6, 7]])
-    # In [3]: np.mean(arr, axis=1)
-    # Out[2]: array([3., 6.])
-    #
-    # You may be be able to solve this in four lines, without a loop.
-    # But solve it any way you can.
-    # This is a placeholder, replace it to write your solution.
+
+    data = img.get_fdata()
+    # create two timeseries with n-1 volumes and find the difference between them
+    # remove the last volume
+    img_start = np.delete(data, slice(data.shape[-1]-1,data.shape[-1]), axis=3) 
```

Comment:
From previous comment (I accidentally closed), suggested change
```suggestion
    img_start = data[...,:-1]
```